src/starling/core/Starling.hx

```diff
@@ -460,15 +459,15 @@ class Starling extends EventDispatcher
 			cast Context3DStencilAction.DECREMENT_SATURATE
 		);
 		
-		var renderModeStr:String = "";
 		
+		//var renderModeStr:String = "";
 		//if (tempRenderMode == Context3DRenderMode.AUTO) renderModeStr = "auto";
 		//else if (tempRenderMode == Context3DRenderMode.SOFTWARE) renderModeStr = "software";
 		
-		var auto:String = "auto";
+		/*var auto:String = "auto";
 		#if js
 			auto = Context3DRenderMode.AUTO.getName();
-		#end
+		#end*/
```

Comment:
I'd prefer not. I didn't write this code. Maybe it can be useful someday. Comments are stripped out anyway when compiled